### PR TITLE
Remove projectiles of a specific entity

### PR DESCRIPTION
**Pull request by hettoo**
For race it would be nice to be able to remove projectiles associated with a specific entity.

At the moment players can abuse weapons by shooting at the spawn area and then respawning.

### CODE REVIEW

File: source/game/g_gametypes.cpp [hunk 1]
```diff
@@ -1368,6 +1368,22 @@ void G_Match_ToggleReady( edict_t *ent )
 /*
 * G_Match_RemoveAllProjectiles
```
Comment (viciious) [inline source/game/g_gametypes.cpp]: Wrong header?

Comment (hettoo) [inline source/game/g_gametypes.cpp]: Oops :)
Nice catch. I'll update.

To be honest I have no idea what these headers are for. :p

**Comment by viciious** [comment]
Looks fine to me now.

**Comment by hettoo** [comment]
Thanks

**Pull request merged**